Add txt manual verification

mjmayer opened the pull request:
Adds a section to use dig to verify the TXT record used to verify domain ownership.

## Screenshot of section
![image](https://user-images.githubusercontent.com/5890769/148652079-bd1b81aa-4b3a-4be2-b809-7a944d3a6a53.png)

ruffrey commented:
I think we disagreed about this in the past. But throwing it out there. It’s unnecessarily confusing that the screenshot shows a msdc.co subdomain which doesn’t need/have domain validation. Please consider using a normal custom domain.

mjmayer commented:
I've replaced all references to the msdc domain in this PR with `example.com`. I have updated all screenshots. Documentation on how to generate the screenshots has been added to the internal wiki.
